### ISSUE TITLE
Pin RuneLite "plugin" by default

### ISSUE THREAD

**Issue by Kamielvf** [runelite/runelite#6256]
This is to help people who start using the client find the general RuneLite settings more easily. As the list is quite long and a lot of people tend to think RuneLite settings is referring to the plugin sidebar and not to the RuneLite config entry in the list.

**Comment by psikoi** [comment]
I think it should be called General Settings or something too btw

**Comment by pavan-p** [comment]
Why not just make a [settings button](https://i.imgur.com/7a3hvTE.png) on the title bar? Instead of considering it as a plugin.

**Comment by deathbeam** [comment]
We already have settings button

**Comment by Hydrox6** [comment]
@pavan-p that just splits the settings up between two places, not a good idea.

**Comment by Adam-** [comment]
I would prefer to see it permanently pinned at the top of the settings panel.

**Comment by Hydrox6** [comment]
@Adam- Always visible at the top when scrolling, or just as the first item in the list?

**Comment by tmk2003** [comment]
I also would prefer to see it permanently pinned at the top

**Comment by Adam-** [comment]
Maybe the former unless it could be visibly differentiated some better way, since no one can find it currently.

**Comment by deathbeam** [comment]
They cant find it because they dont understand what are we referring to because the name is confusing.

**Comment by Hydrox6** [comment]
Yeah, RuneLite settings in the settings for RuneLite is a bit weird. Renaming it to `General` or `System` would probably work better

**Comment by tmk2003** [comment]
@Adam- I have this started. If this will always be pinned, should I remove the pinning star for it to remove confusion of its purpose or no?

**Comment by Kamielvf** [comment]
@tmk2003 I would remove  the pinning star for it yes, since it will always be at the top. Possibly make it so that its not part of the list so that when scrolling the general settings stay at the top.

**Comment by deathbeam** [comment]
I would keep pinning star so you could unpin it if you dont want it, no reason to not have that option

**Comment by tmk2003** [comment]
@deathbeam that differs from what I thought @Adam- was saying, where it is always pinned.

**Comment by deathbeam** [comment]
Yes, this is my opinion not Adam's

**Comment by Nightfirecat** [comment]
For what it's worth, I'm also of the opinion that pinning it by default would be enough, not permanently pinning it.

**Comment by tmk2003** [comment]
Are we changing the name from RuneLite for the plugin, or keeping it RuneLife for now
